# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/streams/idlharness.any.js.ini
# Commit: fd13db3ea2b3
# Full Hash: fd13db3ea2b3bc2c96ef541680dd44f61aa2ce30
# Author: Tom Schuster <evilpies@gmail.com>
# Date: 2022-12-01 16:18:29
# Regressor Bug: 1734244
# File Overlap Count: 1
# Description:
#   Bug 1734244 - Implement async iteration of ReadableStream. r=mgaudet,peterv
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D153312
# ==============================================================================

diff -r d7c2c8133258 -r fd13db3ea2b3 testing/web-platform/meta/streams/idlharness.any.js.ini
--- a/testing/web-platform/meta/streams/idlharness.any.js.ini	Thu Dec 01 06:11:07 2022 +0200
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,26 +0,0 @@
-[idlharness.any.serviceworker.html]
-  expected:
-    if (os == "android") and fission: [OK, TIMEOUT]
-  [ReadableStream interface: async iterable<any>]
-    expected: FAIL
-
-
-[idlharness.any.html]
-  expected:
-    if (os == "android") and fission: [OK, TIMEOUT]
-  [ReadableStream interface: async iterable<any>]
-    expected: FAIL
-
-
-[idlharness.any.worker.html]
-  expected:
-    if (os == "android") and fission: [OK, TIMEOUT]
-  [ReadableStream interface: async iterable<any>]
-    expected: FAIL
-
-
-[idlharness.any.sharedworker.html]
-  expected:
-    if (os == "android") and fission: [OK, TIMEOUT]
-  [ReadableStream interface: async iterable<any>]
-    expected: FAIL